app/notifiers/course/assessment/submission_question/comment_notifier.rb

```diff
@@ -5,7 +5,7 @@ def post_replied(user, post)
 
     activity = create_activity(actor: user, object: post, event: :replied)
     post.topic.subscriptions.includes(:user).each do |subscription|
-      activity.notify(subscription.user, :email) unless subscription.user == user
+      activity.notify(subscription.user, :email, 'no_greeting_mailer') unless subscription.user == user
```

Comment:
What I might suggest is to instead overwrite `email_user` to adopt the `no greeting mailer` layout path in this notifier. This will reduce the need to change the method partners in `ActivityWrapper` and `Notifier::Base`. Thoughts?

Reply:
That's much neater 👍